spec/lang/well-formed.md

```diff
@@ -199,6 +224,7 @@ impl Constant {
             (Constant::GlobalPointer(relocation), Type::Ptr(_)) => {
                 relocation.check_wf(prog.globals)?;
             }
+            // Question: do we not care about the pointer type?
             (Constant::FnPointer(fn_name), Type::Ptr(_)) => {
```

Comment:
We don't care very strongly, no. But we can require this to be a function pointer, sure -- why not.